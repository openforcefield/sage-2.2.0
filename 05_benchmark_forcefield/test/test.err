Converting records to molecules:   0%|          | 0/223 [00:00<?, ?it/s]Converting records to molecules:   0%|          | 1/223 [00:00<00:34,  6.48it/s]Converting records to molecules:   1%|          | 2/223 [00:00<00:36,  6.05it/s]Converting records to molecules:   2%|▏         | 4/223 [00:00<00:24,  9.01it/s]Converting records to molecules:   3%|▎         | 6/223 [00:00<00:18, 12.00it/s]Converting records to molecules:   4%|▎         | 8/223 [00:00<00:18, 11.70it/s]Converting records to molecules:   4%|▍         | 10/223 [00:00<00:16, 12.85it/s]Converting records to molecules:   5%|▌         | 12/223 [00:01<00:19, 10.76it/s]Converting records to molecules:   6%|▋         | 14/223 [00:01<00:21,  9.92it/s]Converting records to molecules:   7%|▋         | 16/223 [00:01<00:18, 11.38it/s]Converting records to molecules:   8%|▊         | 18/223 [00:01<00:19, 10.35it/s]Converting records to molecules:   9%|▉         | 20/223 [00:01<00:17, 11.50it/s]Converting records to molecules:  10%|▉         | 22/223 [00:01<00:16, 12.23it/s]Converting records to molecules:  11%|█         | 24/223 [00:02<00:16, 12.22it/s]Converting records to molecules:  12%|█▏        | 26/223 [00:02<00:15, 12.54it/s]Converting records to molecules:  13%|█▎        | 29/223 [00:02<00:13, 14.46it/s]Converting records to molecules:  14%|█▍        | 31/223 [00:02<00:15, 12.57it/s]Converting records to molecules:  15%|█▍        | 33/223 [00:02<00:17, 10.81it/s]Converting records to molecules:  16%|█▌        | 35/223 [00:03<00:16, 11.11it/s]Converting records to molecules:  17%|█▋        | 37/223 [00:03<00:14, 12.60it/s]Converting records to molecules:  17%|█▋        | 39/223 [00:03<00:15, 11.97it/s]Converting records to molecules:  18%|█▊        | 41/223 [00:03<00:19,  9.18it/s]Converting records to molecules:  19%|█▉        | 43/223 [00:03<00:17, 10.02it/s]Converting records to molecules:  20%|██        | 45/223 [00:04<00:17, 10.32it/s]Converting records to molecules:  21%|██        | 47/223 [00:04<00:15, 11.08it/s]Converting records to molecules:  22%|██▏       | 49/223 [00:04<00:13, 12.71it/s]Converting records to molecules:  23%|██▎       | 52/223 [00:04<00:11, 14.48it/s]Converting records to molecules:  24%|██▍       | 54/223 [00:04<00:12, 13.11it/s]Converting records to molecules:  25%|██▌       | 56/223 [00:04<00:11, 14.07it/s]Converting records to molecules:  26%|██▋       | 59/223 [00:04<00:10, 15.44it/s]Converting records to molecules:  27%|██▋       | 61/223 [00:05<00:09, 16.29it/s]Converting records to molecules:  29%|██▊       | 64/223 [00:05<00:11, 13.73it/s]Converting records to molecules:  30%|██▉       | 66/223 [00:05<00:11, 14.19it/s]Converting records to molecules:  30%|███       | 68/223 [00:05<00:10, 14.85it/s]Converting records to molecules:  31%|███▏      | 70/223 [00:05<00:10, 14.23it/s]Converting records to molecules:  32%|███▏      | 72/223 [00:05<00:11, 13.59it/s]Converting records to molecules:  33%|███▎      | 74/223 [00:06<00:15,  9.79it/s]Converting records to molecules:  34%|███▍      | 76/223 [00:06<00:14, 10.49it/s]Converting records to molecules:  35%|███▍      | 78/223 [00:06<00:13, 10.88it/s]Converting records to molecules:  36%|███▌      | 80/223 [00:06<00:16,  8.72it/s]Converting records to molecules:  37%|███▋      | 82/223 [00:07<00:15,  9.07it/s]Converting records to molecules:  38%|███▊      | 84/223 [00:07<00:13, 10.58it/s]Converting records to molecules:  39%|███▊      | 86/223 [00:07<00:11, 11.90it/s]Converting records to molecules:  39%|███▉      | 88/223 [00:07<00:10, 13.13it/s]Converting records to molecules:  40%|████      | 90/223 [00:07<00:09, 14.41it/s]Converting records to molecules:  41%|████▏     | 92/223 [00:07<00:10, 12.54it/s]Converting records to molecules:  42%|████▏     | 94/223 [00:07<00:10, 11.99it/s]Converting records to molecules:  43%|████▎     | 96/223 [00:08<00:11, 11.01it/s]Converting records to molecules:  44%|████▍     | 98/223 [00:08<00:10, 11.96it/s]Converting records to molecules:  45%|████▍     | 100/223 [00:08<00:09, 12.87it/s]Converting records to molecules:  46%|████▌     | 102/223 [00:08<00:12,  9.69it/s]Converting records to molecules:  47%|████▋     | 104/223 [00:08<00:10, 10.94it/s]Converting records to molecules:  48%|████▊     | 107/223 [00:09<00:09, 11.90it/s]Converting records to molecules:  49%|████▉     | 109/223 [00:09<00:12,  9.24it/s]Converting records to molecules:  50%|████▉     | 111/223 [00:09<00:12,  8.97it/s]Converting records to molecules:  51%|█████     | 113/223 [00:09<00:11,  9.29it/s]Converting records to molecules:  52%|█████▏    | 115/223 [00:10<00:10, 10.04it/s]Converting records to molecules:  52%|█████▏    | 117/223 [00:10<00:14,  7.31it/s]Converting records to molecules:  53%|█████▎    | 118/223 [00:10<00:13,  7.52it/s]Converting records to molecules:  54%|█████▍    | 120/223 [00:10<00:11,  9.00it/s]Converting records to molecules:  55%|█████▍    | 122/223 [00:10<00:09, 10.67it/s]Converting records to molecules:  56%|█████▌    | 124/223 [00:11<00:08, 11.16it/s]Converting records to molecules:  57%|█████▋    | 126/223 [00:11<00:07, 12.52it/s]Converting records to molecules:  57%|█████▋    | 128/223 [00:11<00:07, 12.29it/s]Converting records to molecules:  58%|█████▊    | 130/223 [00:11<00:06, 13.84it/s]Converting records to molecules:  59%|█████▉    | 132/223 [00:11<00:06, 14.72it/s]Converting records to molecules:  60%|██████    | 134/223 [00:11<00:05, 15.86it/s]Converting records to molecules:  61%|██████▏   | 137/223 [00:11<00:05, 16.58it/s]Converting records to molecules:  62%|██████▏   | 139/223 [00:12<00:07, 11.34it/s]Converting records to molecules:  63%|██████▎   | 141/223 [00:12<00:06, 11.79it/s]Converting records to molecules:  65%|██████▍   | 144/223 [00:12<00:05, 14.17it/s]Converting records to molecules:  65%|██████▌   | 146/223 [00:12<00:05, 14.51it/s]Converting records to molecules:  66%|██████▋   | 148/223 [00:12<00:07, 10.18it/s]Converting records to molecules:  67%|██████▋   | 150/223 [00:13<00:08,  9.06it/s]Converting records to molecules:  68%|██████▊   | 152/223 [00:13<00:07,  9.97it/s]Converting records to molecules:  69%|██████▉   | 154/223 [00:13<00:05, 11.61it/s]Converting records to molecules:  70%|██████▉   | 156/223 [00:13<00:05, 11.50it/s]Converting records to molecules:  71%|███████   | 158/223 [00:13<00:05, 11.60it/s]Converting records to molecules:  72%|███████▏  | 160/223 [00:14<00:06,  9.69it/s]Converting records to molecules:  73%|███████▎  | 162/223 [00:14<00:08,  7.24it/s]Converting records to molecules:  74%|███████▎  | 164/223 [00:14<00:06,  8.81it/s]Converting records to molecules:  74%|███████▍  | 166/223 [00:14<00:06,  9.21it/s]Converting records to molecules:  75%|███████▌  | 168/223 [00:14<00:05, 10.60it/s]Converting records to molecules:  76%|███████▌  | 170/223 [00:15<00:04, 12.29it/s]Converting records to molecules:  78%|███████▊  | 173/223 [00:15<00:03, 14.35it/s]Converting records to molecules:  78%|███████▊  | 175/223 [00:15<00:03, 12.57it/s]Converting records to molecules:  79%|███████▉  | 177/223 [00:15<00:03, 12.32it/s]Converting records to molecules:  80%|████████  | 179/223 [00:15<00:03, 12.29it/s]Converting records to molecules:  81%|████████  | 181/223 [00:15<00:03, 11.45it/s]Converting records to molecules:  82%|████████▏ | 183/223 [00:16<00:03, 10.52it/s]Converting records to molecules:  83%|████████▎ | 185/223 [00:16<00:03, 10.35it/s]Converting records to molecules:  84%|████████▍ | 187/223 [00:16<00:03, 11.22it/s]Converting records to molecules:  85%|████████▍ | 189/223 [00:16<00:03, 11.08it/s]Converting records to molecules:  86%|████████▌ | 191/223 [00:17<00:04,  7.24it/s]Converting records to molecules:  87%|████████▋ | 193/223 [00:17<00:03,  8.48it/s]Converting records to molecules:  88%|████████▊ | 196/223 [00:17<00:02, 10.32it/s]Converting records to molecules:  89%|████████▉ | 198/223 [00:17<00:02, 11.44it/s]Converting records to molecules:  90%|████████▉ | 200/223 [00:17<00:01, 12.46it/s]Converting records to molecules:  91%|█████████ | 202/223 [00:18<00:01, 11.30it/s]Converting records to molecules:  91%|█████████▏| 204/223 [00:18<00:01, 11.78it/s]Converting records to molecules:  92%|█████████▏| 206/223 [00:18<00:01, 11.89it/s]Converting records to molecules:  93%|█████████▎| 208/223 [00:18<00:01, 12.79it/s]Converting records to molecules:  94%|█████████▍| 210/223 [00:18<00:00, 13.06it/s]Converting records to molecules:  95%|█████████▌| 212/223 [00:18<00:01,  9.41it/s]Converting records to molecules:  96%|█████████▌| 214/223 [00:19<00:01,  8.73it/s]Converting records to molecules:  97%|█████████▋| 216/223 [00:19<00:00,  9.35it/s]Converting records to molecules:  98%|█████████▊| 218/223 [00:19<00:00, 10.99it/s]Converting records to molecules:  99%|█████████▊| 220/223 [00:19<00:00, 12.26it/s]Converting records to molecules: 100%|█████████▉| 222/223 [00:19<00:00, 12.65it/s]Converting records to molecules: 100%|██████████| 223/223 [00:19<00:00, 11.26it/s]
Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:   0%|          | 0/223 [00:00<?, ?it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:   0%|          | 1/223 [00:53<3:17:26, 53.36s/it]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:   1%|▏         | 3/223 [00:53<50:59, 13.91s/it]  Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:   2%|▏         | 5/223 [00:53<24:41,  6.80s/it]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:   3%|▎         | 7/223 [00:53<14:13,  3.95s/it]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:   4%|▍         | 9/223 [00:53<08:51,  2.48s/it]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:   5%|▍         | 11/223 [00:54<05:47,  1.64s/it]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:   6%|▌         | 13/223 [00:54<03:54,  1.12s/it]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:   7%|▋         | 15/223 [00:54<02:44,  1.27it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:   8%|▊         | 17/223 [00:54<01:57,  1.76it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:   9%|▊         | 19/223 [00:54<01:23,  2.44it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:   9%|▉         | 21/223 [00:55<01:06,  3.03it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  10%|█         | 23/223 [00:55<00:49,  4.03it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  12%|█▏        | 26/223 [00:55<00:33,  5.83it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  13%|█▎        | 29/223 [00:55<00:24,  8.01it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  14%|█▍        | 32/223 [00:55<00:19,  9.65it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  15%|█▌        | 34/223 [00:55<00:17, 10.68it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  17%|█▋        | 37/223 [00:55<00:14, 12.83it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  17%|█▋        | 39/223 [00:56<00:13, 13.84it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  18%|█▊        | 41/223 [00:56<00:12, 14.46it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  20%|█▉        | 44/223 [00:56<00:10, 16.40it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  21%|██        | 46/223 [00:56<00:10, 16.12it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  22%|██▏       | 48/223 [00:56<00:10, 16.98it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  22%|██▏       | 50/223 [00:56<00:11, 15.18it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  24%|██▍       | 53/223 [00:56<00:09, 17.10it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  25%|██▍       | 55/223 [00:56<00:09, 17.24it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  26%|██▌       | 58/223 [00:57<00:08, 18.59it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  27%|██▋       | 60/223 [00:57<00:09, 17.89it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  28%|██▊       | 62/223 [00:57<00:09, 16.82it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  29%|██▊       | 64/223 [00:57<00:13, 11.93it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  30%|██▉       | 66/223 [00:57<00:12, 12.41it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  30%|███       | 68/223 [00:57<00:13, 11.70it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  31%|███▏      | 70/223 [00:58<00:12, 12.29it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  32%|███▏      | 72/223 [00:58<00:11, 13.43it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  33%|███▎      | 74/223 [00:58<00:10, 13.86it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  34%|███▍      | 76/223 [00:58<00:10, 13.86it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  35%|███▍      | 78/223 [00:58<00:09, 14.66it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  36%|███▋      | 81/223 [00:58<00:08, 16.40it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  37%|███▋      | 83/223 [00:59<00:12, 10.80it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  38%|███▊      | 85/223 [00:59<00:12, 11.40it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  39%|███▉      | 87/223 [00:59<00:11, 11.85it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  40%|███▉      | 89/223 [00:59<00:10, 12.98it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  41%|████      | 91/223 [00:59<00:10, 12.45it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  42%|████▏     | 93/223 [00:59<00:10, 12.20it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  43%|████▎     | 95/223 [01:00<00:12, 10.18it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  43%|████▎     | 97/223 [01:00<00:10, 11.54it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  44%|████▍     | 99/223 [01:00<00:10, 12.03it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  46%|████▌     | 102/223 [01:00<00:09, 13.25it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  47%|████▋     | 104/223 [01:00<00:08, 13.85it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  48%|████▊     | 106/223 [01:00<00:08, 13.09it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  49%|████▉     | 109/223 [01:01<00:08, 13.87it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  50%|████▉     | 111/223 [01:01<00:08, 13.80it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  51%|█████     | 114/223 [01:01<00:06, 15.83it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  52%|█████▏    | 116/223 [01:01<00:07, 14.26it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  53%|█████▎    | 118/223 [01:01<00:07, 14.95it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  54%|█████▍    | 120/223 [01:01<00:07, 14.52it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  55%|█████▍    | 122/223 [01:01<00:06, 15.56it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  56%|█████▌    | 124/223 [01:02<00:06, 15.39it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  57%|█████▋    | 127/223 [01:02<00:05, 16.44it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  58%|█████▊    | 129/223 [01:02<00:06, 14.33it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  59%|█████▊    | 131/223 [01:02<00:05, 15.52it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  60%|█████▉    | 133/223 [01:02<00:05, 16.07it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  61%|██████    | 135/223 [01:02<00:05, 16.90it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  61%|██████▏   | 137/223 [01:02<00:05, 16.48it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  63%|██████▎   | 140/223 [01:03<00:04, 17.74it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  64%|██████▎   | 142/223 [01:03<00:05, 13.77it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  65%|██████▍   | 144/223 [01:03<00:05, 14.64it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  65%|██████▌   | 146/223 [01:03<00:05, 15.38it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  66%|██████▋   | 148/223 [01:03<00:04, 15.42it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  67%|██████▋   | 150/223 [01:03<00:04, 15.13it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  69%|██████▊   | 153/223 [01:03<00:04, 16.83it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  70%|██████▉   | 155/223 [01:04<00:04, 16.22it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  70%|███████   | 157/223 [01:04<00:07,  8.77it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  71%|███████▏  | 159/223 [01:04<00:06,  9.39it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  72%|███████▏  | 161/223 [01:04<00:05, 10.64it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  73%|███████▎  | 163/223 [01:04<00:05, 11.89it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  74%|███████▍  | 165/223 [01:05<00:05,  9.68it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  75%|███████▌  | 168/223 [01:05<00:04, 11.89it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  76%|███████▌  | 170/223 [01:05<00:04, 13.14it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  77%|███████▋  | 172/223 [01:05<00:03, 14.05it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  78%|███████▊  | 174/223 [01:05<00:03, 15.16it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  79%|███████▉  | 176/223 [01:05<00:03, 15.57it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  80%|███████▉  | 178/223 [01:06<00:02, 16.13it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  81%|████████  | 181/223 [01:06<00:02, 18.31it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  83%|████████▎ | 184/223 [01:06<00:02, 18.13it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  83%|████████▎ | 186/223 [01:06<00:02, 17.89it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  84%|████████▍ | 188/223 [01:06<00:01, 17.53it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  86%|████████▌ | 191/223 [01:06<00:01, 18.69it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  87%|████████▋ | 194/223 [01:06<00:01, 19.47it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  88%|████████▊ | 196/223 [01:06<00:01, 18.60it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  89%|████████▉ | 198/223 [01:07<00:01, 17.92it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  90%|████████▉ | 200/223 [01:07<00:01, 16.68it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  91%|█████████ | 202/223 [01:07<00:01, 17.38it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  92%|█████████▏| 205/223 [01:07<00:00, 19.00it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  93%|█████████▎| 208/223 [01:07<00:00, 17.81it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  95%|█████████▍| 211/223 [01:07<00:00, 19.04it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  96%|█████████▌| 214/223 [01:07<00:00, 20.13it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  97%|█████████▋| 217/223 [01:08<00:00, 20.05it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml:  99%|█████████▊| 220/223 [01:08<00:00, 19.09it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml: 100%|█████████▉| 222/223 [01:08<00:00, 18.59it/s]Building and minimizing systems with openff_unconstrained-2.2.0-rc1.offxml: 100%|██████████| 223/223 [01:08<00:00,  3.26it/s]
Traceback (most recent call last):
  File "/data/homezvol3/amcisaac/miniconda3/envs/ib-env/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1969, in _exec_single_context
    self.dialect.do_execute(
  File "/data/homezvol3/amcisaac/miniconda3/envs/ib-env/lib/python3.10/site-packages/sqlalchemy/engine/default.py", line 922, in do_execute
    cursor.execute(statement, parameters)
sqlite3.OperationalError: no such column: qm_conformers.qcarchive_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/dfs6/pub/amcisaac/sage-2.2.0/05_benchmark_forcefield/benchmark.py", line 58, in <module>
    main()
  File "/data/homezvol3/amcisaac/miniconda3/envs/ib-env/lib/python3.10/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "/data/homezvol3/amcisaac/miniconda3/envs/ib-env/lib/python3.10/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/data/homezvol3/amcisaac/miniconda3/envs/ib-env/lib/python3.10/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/data/homezvol3/amcisaac/miniconda3/envs/ib-env/lib/python3.10/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/dfs6/pub/amcisaac/sage-2.2.0/05_benchmark_forcefield/benchmark.py", line 50, in main
    store.get_dde(forcefield).to_csv(f"{out_dir}/dde.csv")
  File "/dfs6/pub/amcisaac/ib/ibstore/_store.py", line 492, in get_dde
    mm_energies = self.get_mm_energies_by_molecule_id(molecule_id, force_field)
  File "/dfs6/pub/amcisaac/ib/ibstore/_store.py", line 324, in get_mm_energies_by_molecule_id
    .all()
  File "/data/homezvol3/amcisaac/miniconda3/envs/ib-env/lib/python3.10/site-packages/sqlalchemy/orm/query.py", line 2693, in all
    return self._iter().all()  # type: ignore
  File "/data/homezvol3/amcisaac/miniconda3/envs/ib-env/lib/python3.10/site-packages/sqlalchemy/orm/query.py", line 2847, in _iter
    result: Union[ScalarResult[_T], Result[_T]] = self.session.execute(
  File "/data/homezvol3/amcisaac/miniconda3/envs/ib-env/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 2308, in execute
    return self._execute_internal(
  File "/data/homezvol3/amcisaac/miniconda3/envs/ib-env/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 2190, in _execute_internal
    result: Result[Any] = compile_state_cls.orm_execute_statement(
  File "/data/homezvol3/amcisaac/miniconda3/envs/ib-env/lib/python3.10/site-packages/sqlalchemy/orm/context.py", line 293, in orm_execute_statement
    result = conn.execute(
  File "/data/homezvol3/amcisaac/miniconda3/envs/ib-env/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1416, in execute
    return meth(
  File "/data/homezvol3/amcisaac/miniconda3/envs/ib-env/lib/python3.10/site-packages/sqlalchemy/sql/elements.py", line 517, in _execute_on_connection
    return connection._execute_clauseelement(
  File "/data/homezvol3/amcisaac/miniconda3/envs/ib-env/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1639, in _execute_clauseelement
    ret = self._execute_context(
  File "/data/homezvol3/amcisaac/miniconda3/envs/ib-env/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1848, in _execute_context
    return self._exec_single_context(
  File "/data/homezvol3/amcisaac/miniconda3/envs/ib-env/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1988, in _exec_single_context
    self._handle_dbapi_exception(
  File "/data/homezvol3/amcisaac/miniconda3/envs/ib-env/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 2344, in _handle_dbapi_exception
    raise sqlalchemy_exception.with_traceback(exc_info[2]) from e
  File "/data/homezvol3/amcisaac/miniconda3/envs/ib-env/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1969, in _exec_single_context
    self.dialect.do_execute(
  File "/data/homezvol3/amcisaac/miniconda3/envs/ib-env/lib/python3.10/site-packages/sqlalchemy/engine/default.py", line 922, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) no such column: qm_conformers.qcarchive_id
[SQL: SELECT mm_conformers.energy AS mm_conformers_energy 
FROM mm_conformers 
WHERE mm_conformers.parent_id = ? AND mm_conformers.force_field = ? ORDER BY qm_conformers.qcarchive_id]
[parameters: (7, 'openff_unconstrained-2.2.0-rc1.offxml')]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
