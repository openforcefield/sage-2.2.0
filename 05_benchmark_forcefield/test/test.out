Wed Feb 14 16:03:07 PST 2024
hpc3-21-17
loading initial dataset from datasets/OpenFF-Aniline-Para-Opt-v1.0.json
generating database, saving to test.sqlite
started optimizing store
finished optimizing after 74.14609861373901 sec
Wed Feb 14 16:06:09 PST 2024
